### PR TITLE
fix console warning, update leads page 

### PR DESCRIPTION
**Pull request by 7seven7lst**
1. fix console warning,
2. update leads page to conditionally render different content for TermL vs IUL
3. update sidebar color to match the new UI
4. customer page WIP (getting /users api throw a 500 error so wasn't able to retrieve user from backend)

@escoffon

### CODE REVIEW

File: app/client/web/containers/customers/index.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    status: "<TBD>", // missing
+    avatar: "", // missing
+ }
+
+ class OpoCustomersPage extends Component {
+
+  componentDidMount() {
+    const srv = new OpoCoreUserAPIService();
+    srv.show(1)
+	  .then(data => {
```
Comment (7seven7lst) [inline app/client/web/containers/customers/index.jsx]: Here is where the 500 error occurs. I have opo_core_user in my db with id 1

File: app/client/web/containers/customers/index.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ const customerData = {
+    first_name: "First",
+    last_name: "Last",
+    state: "CA", // missing
+    zipcode: "90001", // missing
+    default_phone: "main",
+    phones: {"main":"1234445566"},
+    status: "<TBD>", // missing
+    avatar: "", // missing
+ }
```
Comment (7seven7lst) [inline app/client/web/containers/customers/index.jsx]: to display customer data, we need those attr added.

File: app/client/web/containers/customers/customerCard.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          <Col xs={{ size: 1, offset: 10 }}>
+            <i className="fas fa-pencil-alt"></i>
+          </Col>
+          <Col xs={{ size: 1 }}>
+            <i className="fas fa-times"></i>
+          </Col>
+        </Row>
+        <Row>
+          <Col xs="3">
+            <img className="img-avatar" src="https://placeholdit.imgix.net/~text?txtsize=33&txt=Hi&w=120&h=120" alt="Card image cap" />
```
Comment (7seven7lst) [inline app/client/web/containers/customers/customerCard.jsx]: placeholder image for customer avatar

**Comment by escoffon** [comment]
@7seven7lst I merged the PR so that I can take a look at the 500 (couldn't pick up demo-stan because of local changes)

**Pull request merged**